binsync/decompilers/angr/controller.py

```diff
@@ -103,6 +103,9 @@ def rebase_addr(self, addr, up=False):
     def goto_address(self, func_addr):
         self._workspace.jump_to(self.rebase_addr(func_addr, up=True))
 
+    def save_decompiler_database(self) -> None:
```

Comment:
This can be confusing naming. Can we call it something like `save_native_decompiler_database`, so people know it's not the BS Database. Also decide on a better return value. I suggest either returning `True` or the path to the saved native db. Docstring could help a little as well.